readthedocs/tutorial.rst

```diff
@@ -397,9 +408,7 @@ container, or whatever hostname you specified for it when you ran the ``docker r
       submit --stdout hostname.stdout.txt /bin/hostname
 
       # check to see if the output was written to file /home/xenon/hostname.stdout.txt
-      ssh -p 10022 xenon@localhost ls -l
-      # see what's in it
-      ssh -p 10022 xenon@localhost cat hostname.stdout.txt
+      xenon filesystem sftp --location localhost:10022 --username xenon --password javagat download hostname.stdout.txt -
```

Comment:
I would be in favor of retaining ``ssh`` here, since you're comparing the xenon result against what people already know.

Reply:
Nope. I've just explained to them how to do this a few minutes earlier. Switching back to ssh shows a lack of trust in our own tool.